src/main/java/frc/robot/Robot.java

```diff
@@ -50,20 +51,25 @@ public void robotInit() {
 		System.out.println("robotInit");
 		input = new TeleopInput();
 
-		Compressor pneumaticsCompressor = new Compressor(PneumaticsModuleType.CTREPCM);
-
-		pneumaticsCompressor.enableDigital();
+		//Enable Compressor feedback loop on the REV PH
+		new Compressor(1, PneumaticsModuleType.REVPH).enableDigital();
 
 		// Instantiate all systems here
 		driveFsmSystem = new DriveFSMSystem();
 		ballSystem = new BallHandlingFSM();
 		grabberSystem = new GrabberFSM();
-		UsbCamera rearCam = CameraServer.startAutomaticCapture("Rear Camera", 0);
-		CvSink cvSinkRear = CameraServer.getVideo(rearCam);
-		CvSource outputStreamRear =
-			new CvSource("Rear Camera", PixelFormat.kMJPEG, camWidth, camHeight, fps);
-		cvSinkRear.setSource(outputStreamRear);
-		rearCam.setBrightness(cameraBrightness);
+
+		//Initialize CV resources
+		try {
+			UsbCamera driverCamera = CameraServer.startAutomaticCapture("Driver Camera", 0);
+			CvSink cvSinkRear = CameraServer.getVideo(driverCamera);
+			CvSource outputStreamRear =
+				new CvSource("Driver Camera", PixelFormat.kMJPEG, camWidth, camHeight, fps);
+			cvSinkRear.setSource(outputStreamRear);
+			driverCamera.setBrightness(cameraBrightness);
+		} catch (VideoException e) {
+			e.printStackTrace();
```

Comment:
We could also System.out.println(e) to get the summary string for the exception. But this works too.